TSC-Charter.md

```diff
@@ -50,43 +66,53 @@ to continue attending TSC meetings as an observer.
 
 ## Section 5. Responsibilities of the TSC.  
 
-The TSC is responsible for all technical development within the Planetary Software Organization,
-including:
-
-* Setting release dates.
-* Release quality standards.
-* Technical direction.
-* Project governance and process (including this policy).
-* GitHub repository hosting.
-* Maintaining a list of affiliate projects
-* Conduct guidelines.
-* Maintaining the list of additional Collaborators.
-* Development process and coding standards.
+The TSC exercises autonomy in setting up and maintaining procedures and policies,
+as well as management and administrative structures as it deems appropriate for the
+maintenance and operation of these projects and resources.
+
+The TSC is responsible for all technical development within the
+Planetary Software Organization, including:
+
+* Technical direction
+* Setting and maintaining governance rules for the conduct and make-up of the
+  TSC, Working Groups and other bodies within the TSC's domain
+* Creating new repositories and projects under the _planetarysoftware_ GitHub organization
```

Comment:
maybe make this a link to the planetary software org page on github

Reply:
Fixed.